mobify/platform-scaffold
# Enable webpack build notifications

Because when a webpack build fails, you have to check the terminal to even know it's failed.

![image](https://user-images.githubusercontent.com/77138/27713113-deefa2e2-5cde-11e7-8037-ded54a56d37e.png)

## Dependency impact

The number of modules installed appears to be almost negligible (we already depend on many of the same deps that `webpack-notifier` depends on):

```sh
$ du -d 0 node_modules
1111488	node_modules
$ du -h -d 0 node_modules
543M	node_modules
$ git checkout webpack-notifications
Switched to branch 'webpack-notifications'
Your branch is up-to-date with 'origin/webpack-notifications'.
$ npm install > /dev/null 2>&1
$ du -d 0 node_modules
1117344	node_modules
$ du -h -d 0 node_modules
546M	node_modules
```

 **JIRA**: (link to JIRA ticket)
 **Linked PRs**: (links to corresponding PRs, optional)

## Changes
- (change1)

## Todos

## How to test-drive this PR
- Run `npm run dev`
- Wait till the dev server is ready
- Open a `container.jsx` file and introduce a syntax error and save
- You should see an OSX notification for "Build failed"
- Revert the syntax error and save again
- You should see an OSX notification for "Build succeeded"
